gslib/commands/rpo.py

```diff
@@ -80,6 +81,14 @@
 _set_help_text = CreateHelpText(_SET_SYNOPSIS, _SET_DESCRIPTION)
 _get_help_text = CreateHelpText(_GET_SYNOPSIS, _GET_DESCRIPTION)
 
+GET_COMMAND = GcloudStorageMap(gcloud_command=[
+    'storage', 'buckets', 'list',
+    '--format=value[separator=": "](name.sub("^", "gs://"),'
+    'rpo.yesno(no="None"))'
+],
+                               flag_map={},
+                               supports_output_translation=True)
```

Comment:
Maybe remove this. I think this argument should be removed from the definition itself, as it is not used anywhere. There was a plan to use it at one point, but later we decided that we are not going to support that level of granularity. Maybe file a bug (internally) to remove this value ?

Reply:
Done!